library/src/main/java/com/google/maps/android/collections/MapObjectManager.java

```diff
@@ -24,7 +24,7 @@
 
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
+import java.util.LinkedHashSet;
```

Comment:
Should `mNamedCollections` and `mAllObjects` on lines 43/44 be changed to `LinkedHashMap`s for predictable order as well?

Reply:
I just confirmed that neither of those collections are able to be iterated over, so consistent ordering should never be an issue there. Only `get()`, `put()`, and `remove()` operations are performed on them internally and there's no API that returns these collections to an external caller.

Reply:
Ok, thanks, that looks right.